kinder/pkg/cluster/manager/actions/kubeadm-upgrade.go

```diff
@@ -141,8 +159,14 @@ func kubeadmUpgradeNode(c *status.Cluster, n *status.Node, upgradeVersion *K8sVe
 	// before two actions where required
 	if n.MustKubeadmVersion().AtLeast(constants.V1_15) {
 		// kubeadm upgrade node
+		nodeArgs := []string{
+			"upgrade", "node", fmt.Sprintf("--v=%d", vLevel),
+		}
+		if kustomizeDir != "" {
```

Comment:
I thought this needed to be version at least `V1_16`?

Reply:
Yes, this is tested up in the chain when `KubeadmUpgrade` starts